lib/cdo/rack/attack.rb

```diff
@@ -31,35 +26,35 @@ def write?
     end
   end
 
-  limits(MAX_TABLE_READS_PER_MIN).each do |limit, period|
+  limits(CDO.max_table_reads_per_sec).each do |limit, period|
     throttle("shared-tables/reads/#{period}", :limit => limit, :period => period.seconds) do |req|
       # extract the channel id for get requests to shared tables. Only match full
       # table reads, because createRecord commands get redirected to individual row
       # reads and we don't want those reads to count toward these limits.
-      req.get? && %r{^/v3/shared-tables/([^/]+)/[^/]+$}.match(req.path).try(:[], 0)
+      req.get? && %r{^/v3/shared-tables/([^/]+)/[^/]+$}.match(req.path).try(:[], 1)
```

Comment:
Do I understand correctly that this change from 0 to 1 is the bugfix that uses just the channel ID (the first capture group `1`) instead of the whole url (the entire match `0`)?